readme.md

```diff
@@ -139,8 +139,10 @@ The easiest way to run it with Portainer on Linux is to use Portainer's stacks f
 ### How to run Nextcloud behind a Cloudflare Tunnel?
 Although it does not seems like it is the case but from AIO perspective a Cloudflare Tunnel works like a reverse proxy. So please follow the [reverse proxy documentation](./reverse-proxy.md) where is documented how to make it run behind a Cloudflare Tunnel.
 
-### Strato Linux VPS
-Stratos VPS are not recommended for running AIO, since they crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit.
+### Disrecommended VPS providers
+- Stratos VPS crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit.
+- Hostingers VPS seem to miss a specifc Kernel feature which is required for AIO to run correctly. See [here](https://help.nextcloud.com/t/help-installing-nc-via-aio-on-vps/153956).
+- In return recommended VPS are those that are KVM/non-virtualized as Docker should work best on them.
```

Comment:
WDYT, should we add an additional section about this?

Reply:
Yes, would be better…

Reply:
okay

Reply:
will do in a follow-up